node-process-argv/echo.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-console */
+process.argv.forEach(value => {
+  console.log(value);

```

Comment:
this should only log the message, not including the path to `node` or the path to `echo.js`